TKGame/Level Editor Content/Stages/.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
May want to come up with some way to have a name stored with the map data to differentiate different levels and their purposes.